lib/src/rules/avoid_renaming_method_parameters.dart

```diff
@@ -108,7 +112,8 @@ class _Visitor extends SimpleAstVisitor<void> {
       var paramIdentifier = parameters[i].identifier;
       if (paramIdentifier != null &&
           paramIdentifier.name != parentParameters[i].name) {
-        rule.reportLint(parameters[i].identifier);
+        rule._reportLintForNodeWithDescription(paramIdentifier,
+            "Don't rename parameter '${parentParameters[i].name}' of the overridden method to '${paramIdentifier.name}'.");
```

Comment:
Consider as an alternative "The parameter '{0}' should have the name '{1}' to match the name used in the overridden method.".